### ISSUE TITLE
[BUG] `addRow` and `addRows` types broken in 4.1.0

### ISSUE THREAD

**Issue by bengry** [exceljs/exceljs#1375]
## 🐛 Bug Report
`v4.1.0` of the library broke the [TypeScript] types of `addRow` and `addRows` by [**requiring** a second argument (`styleOption`)](https://github.com/exceljs/exceljs/compare/v4.0.1...v4.1.0#diff-b52768974e6bc0faccb7d4b75b162c99R1196). This is however an [optional argument](https://github.com/exceljs/exceljs/compare/v4.0.1...v4.1.0#diff-1724d4057ec4f981a0e6910631878ad7R337).

Lib version: 4.1.0

## Steps To Reproduce

```ts
const wb = new ExcelJS.Workbook();
const ws = wb.addWorksheet('blort');

ws.addRow([1]); // type error
```

## The expected behaviour:
There should be no compilationn error, since the second argument is optional.

## Possible solution (optional, but very helpful):

```diff
--- a/node_modules/exceljs/index.d.ts
+++ b/node_modules/exceljs/index.d.ts
@@ -1193,12 +1193,12 @@ export interface Worksheet {
 	 * Add a couple of Rows by key-value, after the last current row, using the column keys,
 	 * or add a row by contiguous Array (assign to columns A, B & C)
 	 */
-	addRow(data: any[] | any, styleOption: string): Row;
+	addRow(data: any[] | any, styleOption?: string): Row;

 	/**
 	 * Add multiple rows by providing an array of arrays or key-value pairs
 	 */
-	addRows(rows: any[], styleOption: string): void;
+	addRows(rows: any[], styleOption?: string): void;
```

**Comment by Subhajitdas298** [comment]
Fixed in #1358. Please use direct version instead on the specific version.
@alubbe Please release a minor fix version, as this type of issue seems to be very common, for people.

**Comment by alubbe** [comment]
@guyonroche would you do the honours?

**Comment by KingDarBoja** [comment]
Looking forward for release as I just hit this error today while updating to latest version. 👍

**Comment by Subhajitdas298** [comment]
With new release @bengry @alubbe this can be closed.

**Comment by bengry** [comment]
Looks to be fixed in 4.1.1. Closing.
